gempy/core/interpolator.py

```diff
@@ -421,7 +481,8 @@ def compile_th_fn_formation_block(self, Z_x=None, weights=None, grid=None, value
             print('Level of Optimization: ', theano.config.optimizer)
             print('Device: ', theano.config.device)
             print('Precision: ', self.dtype)
-            print('Number of faults: ', self.additional_data.structure_data.df.loc['values', 'number faults'])
+            print('Number of faults: ',
+                  self.additional_data.structure_data.df.loc['values', 'number faults'])
```

Comment:
helper functions in structure data will simplify code like this

Reply:
Yep